examples/api-samples/src/browser/api-samples-frontend-module.ts

```diff
@@ -24,6 +24,7 @@ import { bindSampleFileWatching } from './file-watching/sample-file-watching-con
 import { bindVSXCommand } from './vsx/sample-vsx-command-contribution';
 
 import '../../src/browser/style/branding.css';
+import { bindSampleViewContainerPart } from './view/sample-view-container-part-contribution';
```

Comment:
minor: add the `import` with previous imports, we generally add the css import on it's own.